### ISSUE TITLE
Update for endpoints

### ISSUE THREAD

**Issue by sadullahgultekin** [bounswe/bounswe2019group6#153]
In the `/users/me` and `/users/profile/{username}` endpoints, we need `follower number, following number, article number, comment number` infos. Can you add them?

**Comment by bwqr** [comment]
This is definitely important request in terms of reducing the number of requests. These fields should be always in the returned data. 

`/users/profile/{username}`, for that endpoint, If the user is authenticated, these should represent his/her information, otherwise, these fields should be null or have default values. In addition to @sadullahgultekin suggestions, I would like to see there being a `is_following` field among them.

**Comment by bwqr** [comment]
Edit: the detailed required fields are already defined in requirements, which is under 1.1.6.* Profile Page section. The returned data should be revised.

**Comment by enozcan** [comment]
@sadullahgultekin Your request is implemented on 7f93e55 on `backend-dev` branch but not deployed yet. The new response is below.

@bwqr Can you please open new issues for your suggestions in order not to gather multiple features in a single issue? 

Closing the issue.

```json
{
    "username": "admin",
    "email": "admin@email.com",
    "latitude": "46.123",
    "longitude": "46.123",
    "roles": [
        "ROLE_ADMIN"
    ],
    "isPrivate": true,
    "followersCount": 1,
    "followingsCount": 1,
    "articlesCount": 0,
    "commentsCount": 0,
    "iban": null
}
```
